model/commitq/commit_queue.go

```diff
@@ -0,0 +1,84 @@
+package commitq
+
+import (
+	"github.com/pkg/errors"
+)
+
+type CommitQueue struct {
+	ProjectID    string   `bson:"project_id"`
```

Comment:
I think it would be more straightforward to make this tag `bson:"_id"`, since there's already an index on the "_id" field, and that is how we will be querying for this.

Reply:
Done.